.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+Are you targeting develop? All PRs should target the develop branch unless otherwise noted.
```

Comment:
Would recommend not including things like this as commented HTML sections.  They tend to bloat the PR especially when trying to write it or edit it, users will need to delete it out or accept the boilerplate that's there.

If you want to ensure something like the PRs are targeting the `develop` branch, it's might be better to make it an explicit task (checkbox) that they have to click so that they are verifying they did check it's targeting `develop`.

For instance, this is an example I use in PR templates. 

```
## Prerequisites
- [ ] I have verified that there are no existing pull requests that would overlap with this pull request.
- [ ] I have verified that I am following the guidelines as outlined in this project's contribution policy
- [ ] I Have verified that this pull request adheres to this project's code of conduct.
- [ ] I have written a descriptive title for this pull request.
- [ ] I have provided appropriate test coverage were applicable.
```

If users haven't checked all boxes, then the PR isn't even reviewed until they do. Obviously adjust to MonoGame specific requirements

Reply:
This looks like a pretty comprehensive list and a good place to start 😄

Reply:
It feels a bit hardcore to me and might prevent contributions. We can likely find something a bit smoother. Like encouraging test coverage but making them optional. Like noting that PRs with tests will be reviewed in priority, and likewise for PRs that are not conflicting and respecting guidelines.

I prefer to encourage with faster reviews than to fence things off.